### PR TITLE
Typos, punctuation and add some structure (cursive and bold)

### PR DESCRIPTION
**Pull request by lilithfeer**

### CODE REVIEW

File: Home.py [hunk 1]
```diff
@@ -57,19 +57,19 @@ def home_page():
     ***
     
     ### 📊 Differential Abundance
-    - Requires a `csv` file produced by `mbarq analyze` command. This file lists log fold changes (LFC) and FDRs for each gene in the library.
+    - Requires a `csv` file produced by `mbarq analyze` command. This file lists log fold changes (LFC) and false discover rates (FDRs) for each gene in the library.
```
Comment (ASintsova) [inline Home.py:60]: Now that I'm reading it, it should actually say adjusted p-values instead of false discovery rates

File: Home.py [hunk 2]
```diff
@@ -47,8 +47,8 @@ def home_page():
     ### 📈 Exploratory Data Analysis
     
     - Requires a `csv` file with barcode counts for each sample. This file is generated by `mbarq count` and `mbarq merge` commands.
-    - Also requires a `csv` file describing the experimental design (sample data file)
-    - This page generates an interactive PCA plot and barcode abundance plots for genes of interest
+    - Also requires a `csv` file describing the experimental design. (sample data file)
```
Comment (ASintsova) [inline Home.py:50]: Agree on the dot, but should be at the end of the line

File: Home.py [hunk 3]
```diff
@@ -57,19 +57,19 @@ def home_page():
...
-    - Requires a `csv` file produced by `mbarq analyze` command. This file lists log fold changes (LFC) and FDRs for each gene in the library.
+    - Requires a `csv` file produced by `mbarq analyze` command. This file lists log fold changes (LFC) and false discover rates (FDRs) for each gene in the library.
     - This page allows you to look at the 'hits' and create heatmaps of LFCs for genes of interest.
 
     ***
     
     ### 🧶 STRING  
-    - Requires a `csv` file produced by `mbarq analyze` command. This file lists log fold changes (LFC) and FDRs for each gene in the library.
-    - This page allows you to perform functional analysis via STRING-db
+    - Requires a `csv` file produced by `mbarq analyze` command. This file lists log fold changes (LFC) and false discover rates (FDRs) for each gene in the library.
```
Comment (ASintsova) [inline Home.py:66]: Same here, we should replace FDRs with 'adjusted p-values'

File: Home.py [hunk 4]
```diff
@@ -57,19 +57,19 @@ def home_page():
...
+    - This page allows you to perform functional analysis via STRING-db.
     ***
     
     ### 🥚 KEGG
-    - Requires a `csv` file produced by `mbarq analyze` command. This file lists log fold changes (LFC) and FDRs for each gene in the library.
-    - This page allows you to visualize the results in the context of KEGG metabolic maps
+    - Requires a `csv` file produced by `mbarq analyze` command. This file lists log fold changes (LFC) and false discover rates (FDRs) for each gene in the library.
```
Comment (ASintsova) [inline Home.py:71]: Same re. FDRs

**Review by ASintsova — CHANGES_REQUESTED** [review]

File: Home.py (hunk 2 above)
Comment (lilithfeer) [reply Home.py:50]:
```suggestion
    - Also requires a `csv` file describing the experimental design (sample data file).

```

File: Home.py (hunk 1 above)
Comment (lilithfeer) [reply Home.py:60]:
```suggestion
    - Requires a `csv` file produced by `mbarq analyze` command. This file lists log fold changes (LFC) and adjusted p-values for each gene in the library.

```

File: Home.py (hunk 3 above)
Comment (lilithfeer) [reply Home.py:66]:
```suggestion
    - Requires a `csv` file produced by `mbarq analyze` command. This file lists log fold changes (LFC) and adjusted p-values for each gene in the library.

```

File: Home.py (hunk 4 above)
Comment (lilithfeer) [reply Home.py:71]:
```suggestion
    - Requires a `csv` file produced by `mbarq analyze` command. This file lists log fold changes (LFC) and adjusted p-values for each gene in the library.

```

**Review by lilithfeer — COMMENTED** [review]
Implemented the suggestions

**Review by ASintsova — APPROVED** [review]

**Pull request merged**